### ISSUE TITLE
Version of MW is not regonized

### ISSUE THREAD

**Issue by GodMod** [RazeSoldier/mwExtUpgrader#10]
DefaultSettings.php does not contain the Version Number anymore.
```
/**
 * MediaWiki version number
 * @since 1.2
 * @deprecated since 1.35; use the MW_VERSION constant instead
 */
$wgVersion = MW_VERSION;
```

**Comment by RazeSoldier** [comment]
Thanks @GodMod, this will fix.

**Comment by RazeSoldier** [comment]
Fix in adc05a8692.
